api/src/test/java/edu/wpi/first/shuffleboard/api/sources/SingleKeyNetworkTableSourceTest.java

```diff
@@ -1,10 +1,10 @@
-package edu.wpi.first.shuffleboard.sources;
+package edu.wpi.first.shuffleboard.api.sources;
```

Comment:
`SingleKeyNetworkTableSource` should be in `:app`, so this should be moved as well